main.go

```diff
@@ -14,6 +14,7 @@ func main() {
 	logLevelPtr := flag.String("logLevel", "info", "Log level")
 	versionPtr := flag.Bool("version", false, "Display version and exit")
 	debugPtr := flag.Bool("debug", false, "Interactive debug prompts")
+	tagPrefix := flag.String("tagPrefix", "GOAT-IN", "Prefix for GOAT related tags")
```

Comment:
What do you think of naming this `tagPrefixPtr` instead? Only in `main.go`. It'll stay consistent with the others.